src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PropertiesShouldNotBeWriteOnly.cs

```diff
@@ -36,13 +36,45 @@ public abstract class PropertiesShouldNotBeWriteOnlyAnalyzer : DiagnosticAnalyze
                                                                              DiagnosticSeverity.Warning,
                                                                              isEnabledByDefault: false,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/ms182165.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(AddGetterRule, MakeMoreAccessibleRule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterSymbolAction(symbolContext => AnalyzeSymbol(symbolContext), SymbolKind.Property);
+        }
+
+        /// <summary>
+        /// Implementation for CA1044: Properties should not be write only
+        /// </summary>
+        /// <param name="context"></param>
+        private void AnalyzeSymbol(SymbolAnalysisContext context)
+        {
+
+            var property = context.Symbol as IPropertySymbol;
+            // Nothing to do for overwritten issues
+            if (property == null)
+            {
+                return;
+            }
+            // check for when rule is turned off
```

Comment:
Can you clarify the comment - you are not raising a violation if the property was overridden because to fix the error the user will have to fix the base type.